### PR TITLE
Add support for encrypting tokens for tagger, decommissioner, destroyer

### PR DESCRIPTION
**Pull request by fishnix**
* Add support for encrypting client tokens
* Update license to AGPL
* Remove unused Jenkinsfile

### CODE REVIEW

**Comment by damnski** [comment]
lgtm

File: decommission.go [hunk 1]
```diff
@@ -8,27 +8,40 @@ import (
...
 // NewDecommissioner creates a new decommissioning object
-func NewDecommissioner(endpoint, token, id, org string) Decommissioner {
+func NewDecommissioner(endpoint, token, id, org string, encryptToken bool) (Decommissioner, error) {
 	client := &http.Client{Timeout: 30 * time.Second}
+
+	t := token
+	if encryptToken {
+		crytpT, err := bcrypt.GenerateFromPassword([]byte(token), 4)
+		if err != nil {
+			return Decommissioner{}, fmt.Errorf("failed to bcyrpt from password: %s", err)
```
Comment (tenyo) [inline decommission.go:32]: 👀 bcyrpt

File: destroy.go [hunk 2]
```diff
@@ -19,15 +20,26 @@ type Destroyer struct {
...
 // NewDestroyer creates a new destruction object
-func NewDestroyer(endpoint, token, id, org string) Destroyer {
+func NewDestroyer(endpoint, token, id, org string, encryptToken bool) (Destroyer, error) {
 	client := &http.Client{Timeout: 30 * time.Second}
+
+	t := token
+	if encryptToken {
+		crytpT, err := bcrypt.GenerateFromPassword([]byte(token), 4)
+		if err != nil {
+			return Destroyer{}, fmt.Errorf("failed to bcyrpt from password: %s", err)
```
Comment (tenyo) [inline destroy.go:30]: 👀

File: tag.go [hunk 3]
```diff
@@ -20,15 +21,26 @@ type Tagger struct {
...
 // NewTagger creates a new tagging object
-func NewTagger(endpoint, token, id, org string) Tagger {
+func NewTagger(endpoint, token, id, org string, encryptToken bool) (Tagger, error) {
 	client := &http.Client{Timeout: 30 * time.Second}
+
+	t := token
+	if encryptToken {
+		crytpT, err := bcrypt.GenerateFromPassword([]byte(token), 4)
+		if err != nil {
+			return Tagger{}, fmt.Errorf("failed to bcyrpt from password: %s", err)
```
Comment (tenyo) [inline tag.go:31]: 👀

**Review by tenyo — COMMENTED** [review]
Looks good 👍

**Pull request merged**